backend/src/backend/app.py

```diff
@@ -115,6 +131,49 @@ async def get_tasks(user_id: Annotated[str, Depends(get_user_id)]):
     return [{"id": 1, "title": "Buy Milk"}, {"id": 2, "title": "Buy Bread"}]
 
 
+# Open AI chat
+openai.api_key = OPENAI_API_KEY 
+
+
+# Pydantic models for request and response
+class ChatRequest(BaseModel):
+    message: str = Field(..., min_length=1)
+
+
+class ChatResponse(BaseModel):
+    response: str
+
+
+@app.post("/api/chat", response_model=ChatResponse, status_code=status.HTTP_200_OK)
+async def chat(chat_request: ChatRequest):
+    user_message = chat_request.message
+
+    # Prepend instruction to prompt for a brief response
+    prompt = f"Please respond in a brief phrase: {user_message}"
+
+    response = openai.chat.completions.create(
+        model="gpt-3.5-turbo",
+        messages=[
+                {"role": "user", "content": prompt}
+            ],
+        max_tokens=25,  # Limit the number of tokens in the response
+        stop=["\n"],     # Optionally, specify a stop sequence
+    )
+
+    chat_message = response.choices[0].message.content
+    print(chat_message)
+    return ChatResponse(response=chat_message)
+
+
+# override the default status for malformed request payload 422 -> 400
+@app.exception_handler(RequestValidationError)
+async def validation_exception_handler(request: Request, exc: RequestValidationError):
+    return JSONResponse(
+        status_code=status.HTTP_400_BAD_REQUEST,
+        content={"error_message": "Malformed request"}
```

Comment:
I think we should follow this https://fastapi.tiangolo.com/tutorial/handling-errors/#override-request-validation-exceptions

```
@app.exception_handler(RequestValidationError)
async def validation_exception_handler(request, exc):
    return PlainTextResponse(str(exc), status_code=400)
```

With this approach, the FastAPI (Pydantic) validation error will provide more detailed error messages.
I don't feel returning "Malformed request" without details for any type of errors is a good approach

Reply:
I know you're still working on this pull request, but just in case